docs/02-core-concepts/05-compile-targets.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Compiler Targets
+id: compiler-targets
+description: Wing compiler targets and provisioning engines
+keywords: [Wing compiler targets, provisioning engines, multi-cloud]
+---
+
+When working with the Wing programming language, an integral part of the compilation process is the use of compiler targets. In essence, compiler targets dictate the destination and the means for the compiled code. They determine both the cloud environment and the provisioning engine that the code will be deployed to.
+
+## Targets
+
+You can view the list of available compiler targets with the `wing compile -h` command. Here is an example of the output:
+
+```sh
+wing compile -h
+Usage: wing compile [options] <entrypoint>
+
+Compiles a Wing program
+
+Arguments:
+  entrypoint                 program .w entrypoint
+
+Options:
+  -t, --target <target>      Target platform (choices: "tf-aws", "tf-azure", "tf-gcp", "sim", "awscdk", default: "sim")
+  -p, --plugins [plugin...]  Compiler plugins
+  -h, --help                 display help for command
+```
+
+In the Options section, there are several choices for `--target`, such as `tf-aws`, `tf-azure`, `sim`, `awscdk` etc.
+
+It's important to note that these targets contain a combination of provisioning engine and cloud environment in their names. The only exception is `sim`, which is a special target for testing Wing applications locally with the Wing simulator.
+
+## Portability
+
+One of Wing's standout features is its portability. You can write code once and deploy it to any cloud environment that Wing supports. Moreover, Wing is agnostic to the provisioning engine used. Here's an example of a portable Wing code snippet:
+
+```js
+bring cloud;
+
+let bucket = new cloud.Bucket();
+```
+
+### Provisioning engines
+
+When compiling your Wing code for deployment into the cloud, it's crucial to select the right provisioning engine. Provisioning is the process of setting up and creating infrastructure, and the provisioning engine is the driver behind this deployment. Common engines used in the Wing compilation process include Terraform and AWS CDK, with more on the horizon.
+
+Knowing this, it's clear to see how targets represent both the provisioning engine and the cloud destination. For instance, `tf-aws` signifies the use of Terraform for AWS deployment, while `tf-gcp` indicates Terraform for Google Cloud.
+
+## Code to a specific target
+
+There might be times when you need to write code that is specific to a compiler target. For instance, you may want to activate a verbose logging service only when testing locally to save on CloudWatch log storage costs.
```

Comment:
(trying to avoid AWS bias)
```suggestion
There might be times when you need to write code that is specific to a compiler target. For instance, you may want to activate a verbose logging service only when testing locally to save on cloud log storage costs.
```